stl/inc/ranges

```diff
@@ -1947,7 +1947,7 @@ namespace ranges {
                 return _Current;
             }
             _NODISCARD constexpr iterator_t<_Base> base() && noexcept(
-                is_nothrow_move_constructible_v<iterator_t<_Base>>) /* strengthened */ {
+                is_nothrow_move_constructible_v<iterator_t<_Base>>) requires forward_range<_Vw> {

```

Comment:
So I was wonderung why this fails in the `transform_view` tests. You are in the wrong view here. 

This is the `transform_view::iterator` and not the `lazy_split_view::inner_iterator`. Please revert this change here.

You can find those functions as line 3660-3667